### DISCUSSION TITLE
is it production ready?

### DISCUSSION THREAD

**Discussion by Grabber** [nanomq/nanomq#640] [Q&A]

**Comment by JaylinYu** [comment]
Hi Grabber,

As an MQTT broker, in terms of MQTT 3.1.1, NanoMQ 0.3.4 is production-ready. We have a detailed benchmark report for all MQTT scenarios, we are going to make it public very soon. If you need we can share it with you now.

**Comment by Grabber** [comment]
@JaylinYu, could you please share?

I'm in love with the NanoMQ idea, I was always looking for a lighweight MQTT broker and using nng as the networking core is the best move ever made.
